tools/gfx-unit-test/format-unit-tests.cpp

```diff
@@ -337,6 +337,22 @@ namespace gfx_test
                 Slang::makeArray<float>(1.0f, 0.0f, 0.5f, 0.25f));
         }
 
+        // Ignore this test since R64 has large limitations on use, non storage buffer use is a limitation.
+        if(false)
+        {
+            uint32_t texData[] = { 255u, 0u, 0u, 255u,
+                                   255u, 255u, 127u, 127u };
+            ITextureResource::SubresourceData subData = { (void*)texData, 16, 0 };
+
+            auto texView = createTexView(device, size, gfx::Format::R64_UINT, &subData);
+            setUpAndRunTest(device, texView, uintBufferView, "copyTexUint5");
+            compareComputeResult(
+                device,
+                uintResults,
+                Slang::makeArray<uint32_t>(255u, 0u, 0u, 255u,
+                                           255u, 255u, 127u, 127u));
+        }
+
```

Comment:
You will need to either remove this or make it working.

Reply:
I can't make the test working because the test uses a type that does not support uniform (non storage) use

the test is here for the same reason `auto texView = createTexView(device, size, gfx::Format::R32G32B32_FLOAT, &subData);` is added, neither are allowed to work with the resources in the test, but was added for completion(?)

Reply:
I will remove the code, the comment would likely rot as we wait for hardware to eventually support non storage 64bit (which is an if, rather than a when since the need is not really there).